src/components/UI/navbar/index.js

```diff
@@ -55,6 +55,9 @@ const Navbar = () => {
             setUserData({
               userName: responseJson.username,
               firstName: responseJson.first_name,
+              profilePicture: responseJson.picture
+                ? responseJson.picture.url
```

Comment:
```suggestion
                ? responseJson.picture.url || DEFAULT_AVATAR
```
I think this will work

Reply:
I think this will only work when the request fails and there is no response object, what do you think ??

Reply:
If the request fails then this code will never get executed because it won't enter the if block right?

Reply:
Ok not exactly failure of request, but it may happen that the data from backend changes somehow and there is no URL property. So just to be safe from breaking of site, we use optional chaining.

Reply:
Ok I need to read about optional chaining first.